.github/workflows/check_pr.yml

```diff
@@ -49,16 +49,16 @@ jobs:
       matrix:
         os: [macos-latest, windows-latest, ubuntu-latest]
         # node versions to run tests on
-        nodeRun: [20]
+        nodeRun: [22]
         # node version on which code was built and should be tested
-        nodeBuild: [20]
+        nodeBuild: [22]
         include:
           - os: ubuntu-latest
-            nodeRun: 18
-            nodeBuild: 20
+            nodeRun: 20
+            nodeBuild: 22
           - os: ubuntu-latest
             nodeRun: 22
-            nodeBuild: 20
+            nodeBuild: 22
```

Comment:
We should keep node@18 since it's not end-of-life yet. The other test should use Node#20.